modules/host/host_consistency.go

```diff
@@ -0,0 +1,86 @@
+package host
+
+import (
+	"errors"
+)
+
+// consistency.go contians a bunch of consistency checks for the host. Because
```

Comment:
one of many most likely.

I'll go through with spellcheck when the PR is in a later stage. I'm still adding a lot of comments and changing stuff around, not too worried about typos at this point.

Though, I guess it's good to call them as you see them, lest they get missed on the next pass.